tensorflow/python/kernel_tests/clip_ops_test.py

```diff
@@ -369,6 +370,17 @@ def testClipByGlobalNormZero(self):
     self.assertAllClose(np_ans_0, tf_ans_1)
     self.assertAllClose(np_ans_1, tf_ans_2)
 
+  def testClipByGlobalNormInf(self):
+    with self.test_session(use_gpu=True):
+      x0 = constant_op.constant([-2.0, 0.0, np.inf, 4.0, 0.0, 0.0],
+                                shape=[2, 3])
+      x1 = constant_op.constant([1.0, -2.0])
+      clip_norm = 6.0
+
+      _, norm = clip_ops.clip_by_global_norm([x0, x1], clip_norm)
+      with self.assertRaisesRegexp(errors.InvalidArgumentError, "use_norm"):
+        norm.eval()
```

Comment:
Could you add a check to verify that the exception is also triggered when fetching one of the clipped tensors (but not the norm)?